portfolio/src/main/java/com/google/sps/servlets/HomeServlet.java

```diff
@@ -0,0 +1,44 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import java.util.HashMap;
+import com.google.gson.Gson;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Page that displays login status of the user.*/
+@WebServlet("/home")
+public class HomeServlet extends HttpServlet {
+
+/** Checks the current status and acts accordingly to display the right interface. */
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException
+    {
+        response.setContentType("text/html");
+        HashMap<String,String> loginHash = new HashMap<>();
+        UserService userService = UserServiceFactory.getUserService();
+        if (userService.isUserLoggedIn())
```

Comment:
1. There are duplicated statements inside if then else.
For example lines 30, 42. Even #29 and #37 could be using one liner.
2. Var userEmail is never used.
3. Lens 27,28 could be one line: 
 String logoutUrl = userService.createLogoutURL("/");
4. The same for 35,36 .